src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
Nope, it didn't help
![snapshot8](https://cloud.githubusercontent.com/assets/2544251/9098624/f20f8c00-3bc2-11e5-818b-6eaaca5209e9.png)